packages/usa-link/src/usa-link.twig

```diff
@@ -2,7 +2,13 @@
 
 <p>This is <a class="usa-link usa-color-text-visited" href="javascript:void(0);">a visited link</a>.</p>
 
-<p>This is a link that goes to an <a class="usa-link usa-link--external" href="https://i.giphy.com/media/WPzQF6ruiIIVzHNlwX/source.gif">external website</a>.</p>
+<p>
+  This is a link that opens in <em>current</em> tab and goes to an <a class="usa-link usa-link--external" href="https://i.giphy.com/media/WPzQF6ruiIIVzHNlwX/source.gif">external website</a>.
+</p>
+
+<p>
+  This is a link that opens in a <em>new</em> tab and goes to an <a class="usa-link usa-link--external" rel="noreferrer noopener" target="_blank"  href="https://i.giphy.com/media/WPzQF6ruiIIVzHNlwX/source.gif">external website</a>.
```

Comment:
💡 I've explicitly added `rel="noreferrer noopener"` as a best practice; even though most modern browsers implicitly add it.

![image](https://user-images.githubusercontent.com/3385219/230659388-bd854d68-9d25-4e6c-8167-666ffa4c5474.png)
[MDN Browser compatibility →](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility)

Reply:
Since this is a best practice, would it be beneficial to include guidance on `noreferrer noopener` in the documentation?

Reply:
I've kept `noreferrer` since only one is needed in [e348c59], but it might not be needed at all since modern browsers add it implicitly.